react/AppLinker/index.deprecated.spec.jsx

```diff
@@ -84,64 +86,68 @@ describe('app icon', () => {
   })
 
   it('should render correctly', () => {
-    const root = shallow(<DeprecatedAppItem t={tMock} app={app} />).dive()
-    expect(root.getElement()).toMatchSnapshot()
+    const { container } = setup({ app })
+    expect(container).toMatchSnapshot()
   })
 
-  it('should work for native -> native', () => {
-    const root = shallow(
-      <DeprecatedAppItem t={tMock} app={app} onAppSwitch={appSwitchMock} />
-    ).dive()
-    root.find('a').simulate('click')
+  it('should work for web -> web', async () => {
+    isMobileApp.mockReturnValue(false)
+    const { container, user } = setup({ app, onAppSwitch: appSwitchMock })
+    const link = container.querySelector('a')
+    await user.click(link)
     expect(appSwitchMock).not.toHaveBeenCalled()
+    expect(startApp).not.toHaveBeenCalled()
+  })
+
+  it('should work for native -> native', async () => {
     isMobileApp.mockReturnValue(true)
-    root.setState({ nativeAppIsAvailable: true })
-    root.find('a').simulate('click')
-    expect(openNativeFromNativeSpy).toHaveBeenCalled()
+    checkApp.mockResolvedValue(true)
+    const { container, user } = setup({ app, onAppSwitch: appSwitchMock })
+    const link = container.querySelector('a')
+    await user.click(link)
+
     expect(startApp).toHaveBeenCalledWith({
       appId: 'io.cozy.drive.mobile',
       name: 'Cozy Drive',
       uri: 'cozydrive://'
     })
     expect(appSwitchMock).toHaveBeenCalled()
+    console.log('FINITO')
```

Comment:
to remove

Reply:
Fixed in https://github.com/cozy/cozy-ui/compare/afdf6b775cfac127955c238b95653c6fd14ca6ae..367be680b4fb6f8f7e96a3fe5507001e5e2d672e

Thanks 👍